Makefile

```diff
@@ -7,9 +7,15 @@ SPK_BUILD = "005"
 
 all: tailscale-amd64 tailscale-386 tailscale-arm64 tailscale-arm
 
+all-release: tailscale-release-amd64 tailscale-release-386 tailscale-release-arm64 tailscale-release-arm
+
+tailscale-release-%:
+	@./build-package.sh ${TAILSCALE_TRACK} ${TAILSCALE_VERSION} $* ${SPK_BUILD} "6" "false"
+	@./build-package.sh ${TAILSCALE_TRACK} ${TAILSCALE_VERSION} $* ${SPK_BUILD} "7" "true"
+
 tailscale-%:
-	@./build-package.sh ${TAILSCALE_TRACK} ${TAILSCALE_VERSION} $* ${SPK_BUILD} "6"
-	@./build-package.sh ${TAILSCALE_TRACK} ${TAILSCALE_VERSION} $* ${SPK_BUILD} "7"
+	@./build-package.sh ${TAILSCALE_TRACK} ${TAILSCALE_VERSION} $* ${SPK_BUILD} "6" "false"
```

Comment:
maybe instead of "true"/"false" this could be ""/"priv", so it's clearer what's going on

Reply:
How's this?